specs/FamilySafety.md

```diff
@@ -0,0 +1,103 @@
+Family Safety
+===
+
+# Background
+Provide end evelolper a new API to toggle Family Safety feature on and off. End developer can use this API to enable and disable the Family Safety feature. Also provided a iframe filter toggle API to ensure iframes also go through Family Safety services for all subframe navigations if the developer chose to enable iframe support.
+
+# Examples
+## WinRT and .NET   
+```c#
+void FamilySafetyFeatureCmdExecuted(object target, ExecutedRoutedEventArgs e)
+{
+    webView.IsFamilySafetyFeatureEnabled = !webView.IsFamilySafetyFeatureEnabled;
```

Comment:
We usually try not to put product names into API names, because product names can end up changing. If the Family Safety team were to rename, is there a more general name we could use in this API that would still apply?

Reply:
Family Safety is a Microsoft feature. This most likely won't get changed? https://www.microsoft.com/en-us/microsoft-365/family-safety